recipes/xz_utils/cmake/conanfile.py

```diff
@@ -57,6 +57,7 @@ def source(self):
 
     def generate(self):
         tc = CMakeToolchain(self)
+        tc.variables["BUILD_TESTING"] = False
```

Comment:
@Ahajha Why do you need it? The upstream is False by default and we always re-check CMakeList.txt when bumping a version. Did you find any bug? 

https://github.com/tukaani-project/xz/blob/v5.6.0/CMakeLists.txt#L1975

Reply:
@uilianries This is one of those weird built-in CMake variables that's automatically enabled when including the `CTest` module: https://cmake.org/cmake/help/latest/module/CTest.html

Reply:
Not strictly necessary, but I noticed that the tests were being built when making the package, I figured it would be a good idea to skip those.